unicore/ask/service/questions_api.py

```diff
@@ -20,9 +21,35 @@ class QuestionResource(object):
     def __init__(self, request):
         self.request = request
 
-    @view(renderer='json')
+    @view(renderer='json', schema=QuestionSchema)
```

Comment:
Yes it is. [cornice](http://cornice.readthedocs.org/en/latest/validation.html#using-colander) wraps the validation errors in an errors dict. It also returns the deserialized content in `request.validated`